### ISSUE TITLE
Tree-shaking support. Breaking change

### ISSUE THREAD

**Issue by mvsmal** [mvsmal/fable-material-ui#43]
Currently, the structure of Fable.MaterialUI contains 2 types for each MaterialUI Component, a "raw" `ComponentClass` and more handy to use `ReactElement`.
Example:
```fsharp
let inline materialEl<'P when 'P :> IHTMLProp>
    (a:Fable.Import.React.ComponentClass<'P>) (b: IHTMLProp list) c : Fable.Import.React.ReactElement =
    Fable.Helpers.React.from a (keyValueList CaseRules.LowerFirst b |> unbox) c

let AppBar = importDefault<Fable.Import.React.ComponentClass<IHTMLProp>> "@material-ui/core/AppBar"
let inline appBar b c = materialEl AppBar b c
```

With this approach, there are imports of all components, so tree-shaking doesn't help. There is a way to optimize it, but then I need to remove the imports of "raw" `ComponentClass`es, which means a breaking change.

"Raw" `ComponentClass` might be needed for example to pass to [`ReactType`](https://github.com/fable-compiler/fable-react/blob/master/src/Fable.React/Fable.Import.React.fs#L9-L10) DU.
Example:
```fsharp
MaterialProp.Component (AppBar |> U3.Case2)
```

Basically you will be able to import it yourself, if needed. So I would prefer a breaking changing in favor to bundle optimization.

**Comment by mvsmal** [comment]
So it's gonna be a breaking change and will be published with version 3.

**Comment by mvsmal** [comment]
Version 3 released
